src/autoload.php

```diff
@@ -0,0 +1,65 @@
+<?php // phpcs:disable WebimpressCodingStandard.PHP.CorrectClassNameCase.Invalid,SlevomatCodingStandard.TypeHints.DeclareStrictTypes.IncorrectWhitespaceBeforeDeclare
+
+declare(strict_types=1);
+
+namespace Laminas\Stdlib;
+
+use function class_alias;
+
+use const PHP_VERSION_ID;
+
+// Laminas\Stdlib\ArrayObject
+class_alias(
+    PHP_VERSION_ID >= 80100
```

Comment:
Can we perhaps rewrite this so that we have one `if` block for `80100`, and then `class_alias()` with `PHP81` vs `Legacy` in each branch (rather than all conditionals here)

Reply:
Why even class aliases instead of conditional definitions / includes?

Reply:
Conditional definitions are prohibited via our CS checker, and are generally fairly brittle. Conditional ncludes are not a great idea as they (a) have filesystem overhead, and (b) can lead to autoloading conflicts when generating an optimized autoloader (as those will scan all files in a tree to create a map of classes to filenames).

Aliases have the benefit of answering to both the target class and the alias, and fulfill both inheritance trees, allowing slipstreaming them in like this.